src/assets/styles/base/_typography.scss

```diff
@@ -98,23 +83,21 @@ a {
 
 strong {
   color: $color-grey-dark;
-  //font-family: $font-family-sans-serif-bold;
-  //font-weight: 700p
 }
 
 figure {
-  margin: $spacing-vertical-lg 0;
+  margin: $spacing-6 0;
 
-  @media screen and (min-width: map-get($bp, 'md')) {
-    margin: $spacing-vertical-lg 0 $spacing-vertical * 3;
+  @media screen and (min-width: map-get($breakpoint, "breakpoint-3")) {
+    margin: $spacing-6 0 $spacing-4 * 3;
```

Comment:
I know you didn't add the `* 3` yourself, but is this something that could be tidied up?